### DISCUSSION TITLE
Renaming of Package/Container

### DISCUSSION THREAD

**Discussion by FreakeyPlays** [community/community#48645] [New to GitHub] [answered]
Hey,

i just renamed my Project from VS-Lab to (Todo App)[https://github.com/FreakeyPlays/todo-app] and now I wanted to do the same with my Package because the Title of my package is still [vs-lab](https://github.com/FreakeyPlays/todo-app/pkgs/container/vs-lab). Is there any way to rename this package? And if so, how would I be able to rename it to todo-app?

I hope somebody can help me, and thanks in advance!

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! I spoke with a teammate more familiar with Packages and unfortunately, the workaround here is to [delete the package](https://docs.github.com/en/rest/packages?apiVersion=2022-11-28#delete-a-package-for-a-user) and push up using the new name.

**Reply by FreakeyPlays** [reply]
Hey! That was what I was trying to avoid, but then I will do that. But Thanks!

**Reply by queenofcorgis** [reply]
@FreakeyPlays I understand - we checked to see if we could do it from the staff side, but that wasn't possible either.

**Reply by FreakeyPlays** [reply]
No Problem. But I appreciate the effort.
